{"name": "Проверка ошибки при отправке запроса с несуществующей парой логин-пароль", "status": "passed", "start": 1717357824105, "stop": 1717357824227, "uuid": "2669a1e2-1e50-4b4a-a3ae-3898afdfd277", "historyId": "0c416a5fb9a43c46e363993a8e3034f4", "testCaseId": "0c416a5fb9a43c46e363993a8e3034f4", "fullName": "tests.test_login_courier.TestLoginCourier#test_failed_login_courier_without_password", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "msknbfb5654inb"}, {"name": "thread", "value": "17632-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}